learn/how-tos/setTimezone.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setting Time Zone for Datetime widget "
+id: "timezone-date-datetime-time"
+---
+
+The setTimezone function allows precise adjustment of time to specific time zones, ensuring accuracy and consistency in date and time displays.
+
+## Using setTimezone for Datetime widget
+
+1. Drag and drop a Datetime widget. [ learn more about Datetime](/learn/app-development/widgets/form-widgets/date-time-datetime/#datetime-properties)
```

Comment:
Test issue: no-space-in-links Spaces inside link text [Context: "[ learn more about Datetime]"]